connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/FunctionTestSuite.scala

```diff
@@ -229,6 +229,18 @@ class FunctionTestSuite extends ConnectFunSuite {
     schema_of_csv("x,y"),
     schema_of_csv(lit("x,y"), Collections.emptyMap()))
   testEquals("to_csv", to_csv(a), to_csv(a, Collections.emptyMap[String, String]))
+  testEquals(
+    "from_xml",
+    from_xml(a, schema),
+    from_xml(a, lit(schema.toDDL)),
+    // from_xml(a, lit(schema.json)),
```

Comment:
@HyukjinKwon @itholic 
This is failing due to a parse error. So I commented it temporarily. The same passes for from_json() above in this test.
Can you please share some pointers?